src/test/java/com/iexec/core/worker/WorkerServiceRealRepositoryTests.java

```diff
@@ -0,0 +1,82 @@
+package com.iexec.core.worker;
```

Comment:
License is missing here

Reply:
Good catch! Please see [d36d49f](https://github.com/iExecBlockchainComputing/iexec-core/pull/602/commits/d36d49fe1a931b514fadcc96b45ab91a861e6e2a).